File: Two Sum/TwoSum.php
<?php

function twoSum($nums, $target) {
    $collection = array();

    foreach($nums as $key => $num) {
        $subtracted = $target - $num;

        if (count($collection)>0 && array_key_exists($subtracted, $collection)) {
            return array($collection[$subtracted], $key);  
        } else {
            $collection[$num] = $key;
        }
    }
}

$nums = [2,7,11,15];
$target = 9;

var_dump (twoSum($nums, $target));

?>

File: Two Sum/TwoSum.py
def twoSum(self, nums: List[int], target: int) -> List[int]:
    for i in range(0, len(nums)):
        remaining_numbers = {}

        for i, n in enumerate(nums):
            if n in remaining_numbers:
                return [i, remaining_numbers[n]]
                
            remaining_numbers[target-n] = i